### ISSUE TITLE
Unsafe lifecycle methods were found within a strict-mode tree

### ISSUE THREAD

**Issue by ulvido** [bsidelinger912/react-tooltip-lite#37]
![2019-03-13 21-20-31 ekran görüntüsü](https://user-images.githubusercontent.com/27722274/54304333-f870bf80-45d5-11e9-9921-3ee77f3e2cd5.png)

I use react with strict mode and tooltip gives me this error on hover. any ideas?

```
<React.StrictMode>
              <Tooltip
                content="REFRESH"
                padding={"6px"}
                arrow={false}
                hoverDelay={50}
              >
                <Button text={"↻"}
                  className={"btn btn-liste-top"}
                  onMouseDown={() => window.location.reload()} />
              </Tooltip>
</React.StrictMode>
```

**Comment by bsidelinger912** [comment]
Hey @ulvido it looks like this is coming from react-minimalist-portal, hopefully it's as simple as updating that, though we can probably get away with updating our peer-dependency on react to 16+ and just use native portals at this point too.  We'll get this addressed before react 17 comes out.

**Comment by bsidelinger912** [comment]
@ulvido this is fixed with v1.9.2
